x/staking/keeper/val_state_change.go

```diff
@@ -256,8 +256,11 @@ func (k Keeper) ApplyAndReturnValidatorSetUpdates(ctx context.Context) (updates
 		}
 	}
 
+	var valUpdates types.ValidatorUpdates
```

Comment:
this can be shorthanded to `valUpdates := types.ValidatorUpdates{Updates: updates}`